integrations/google-adwords-new/HISTORY.md

```diff
@@ -1,3 +1,8 @@
+1.2.0 / 2020-02-18
+==================
+
+  * Introduces a new setting `disableAdPersonalization` to disable collection of remarketing data for users who do not wish to view personalized ads. When this settiing is `false`, Segment will set `allow_ad_personalization_signals` to false. 
```

Comment:
I think this changelog entry doesn't line up with the behavior. When `disableAdPersonalization` is true, we set `allow_ad_personalization_signals` to false.

Reply:
@gpsamson good catch, fixed.